mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -91,6 +94,9 @@ async fn save_my_maker_swap_event(ctx: &MmArc, swap: &MakerSwap, event: MakerSav
 
     if let SavedSwap::Maker(mut maker_swap) = swap {
         maker_swap.events.push(event);
+        if maker_swap.is_success().is_ok() {
+            maker_swap.fetch_and_set_usd_prices().await;
+        }
```

Comment:
`maker_swap.is_success()` can actually return `Ok(false)`. Please check if it's `Ok(true)` exactly using `if let` keywords.